BuildLoader.py

```diff
@@ -1242,7 +1237,8 @@ def cmd_clean(args):
 				print 'Removing %s' % file
 				os.remove(file)
 
-		for cmd in cmds:
+		if os.path.exists(os.path.join (sbl_dir, '.git')):
+			cmd = 'git clean -xdf BaseTools'
 			x = subprocess.call(cmd.split(' '), cwd=sbl_dir)
```

Comment:
'distclean' does not completely remove all generated files in 'gitless' tree, in which what 'git clean' does. This is slightly different behavior compared to a git tree. If we don't plan to fix it the same way, provide a comment to say so.

Reply:
I am not sure if you want to add comment to code or if you want to print some info. And could you give an example of comments you want to add?
The BuildLoader.py will call edk2 script to build edk2 base tool, but it doesn't provide script to clean up them. I think it is fine not to clean up base tool for 'GitLess' case since it has no other base tool source to build.